src/content/community/get-involved/open-research/index.md

```diff
@@ -0,0 +1,413 @@
+
+#### Background reading
+
+[Introduction to Ethereum sharding](/upgrades/shard-chains/)
+
+[Proto-Danksharding notes](https://notes.ethereum.org/@vbuterin/proto_danksharding_faq)
+
+[Bankless Danksharding video](https://www.youtube.com/watch?v=N5p0TB77flM)
+
+[Ethereum Sharding Research Compendium](https://notes.ethereum.org/@serenity/H1PGqDhpm?type=view)
+
+[Danksharding (Polynya)](https://polynya.medium.com/danksharding-36dc0c8067fe)
+
+#### Recent research
+
+[EIP-4844](https://github.com/ethereum/EIPs/pull/5088)
+
+[Vitalik on sharding and data availability sampling](https://hackmd.io/@vbuterin/sharding_proposal)
+
+
+### Hardware
+
+[Running nodes](/developers/docs/nodes-and-clients/run-a-node/) on modest hardware is fundamental to keeping Ethereum decentralized. Therefore, active research into minimizing the hardware requirements to run nodes is an important area of research.
+
+#### Background reading
+
+[Ethereum on ARM](https://ethereum-on-arm-documentation.readthedocs.io/en/latest/)
+
+
+#### Recent research
+
+[ecdsa on FGPAs](https://ethresear.ch/t/does-ecdsa-on-fpga-solve-the-scaling-problem/6738)
+
+## Security
+
+Security is a broad topic that might include spam/scam prevention, wallet security, hardware security, crypto-economic security, bug hunting and testing of applications and client software and key-management. Contributing to knowledge in these areas will help stimulate mainstream adoption.
+
+### Cryptography & ZKP
+
+Zero-knowledge proofs (ZKP) and cryptography are critical for building privacy and security into Ethereum and its applications. Zero-knowledge is a relatively young but fast-moving space with many open research and development opportunities. Some possibilities include developing more efficient implementations of the [Keccak hashing algorithm](https://hackmd.io/sK7v0lr8Txi1bgION1rRpw?view#Overview), finding better polynomial commitments than currently exist or reducing the cost of ecdsa public key generation and signature verification circuits.
+
+#### Background reading
+
+[0xparc blog](https://0xparc.org/blog)
+
+[zkp.science](https://zkp.science/)
+
+[Zero Knowledge podcast](https://zeroknowledge.fm/)
+
+
+#### Recent research
+
+[Recent advance in elliptic curve cryptography](https://ethresear.ch/t/the-ec-fft-algorithm-without-elliptic-curve-and-isogenies/11346)
+
+[Ethresear.ch ZK](https://ethresear.ch/c/zk-s-nt-arks/13)
+
+
+### Wallets
+
+User-wallets can be browser extensions, desktop and mobile apps or smart contracts on Ethereum. There is active research into social recovery wallets that reduce some of the risk associated with individual-user key management. Associated with dveelopment of wallets is research into alternative forms of account abstraction, which is an important area of nascent research.
+
+#### Background reading
+
+[Our Security page](https://ethereum.org/en/security/)
```

Comment:
Fix link: https://ethereum.org/en/contributing/style-guide/#internal-links

Reply:
Also same feedback as earlier re: anchor text - let's update it to something more descriptive, e.g `Introduction to wallet security` vs. `Our Security page`. Let's avoid terms like "Our page" - the external vs. internal link indicators already demonstrate that.